packages/string-templates/src/index.js

```diff
@@ -1,398 +1,40 @@
-const handlebars = require("handlebars")
-const { registerAll, registerMinimum } = require("./helpers/index")
-const processors = require("./processors")
-const { atob, btoa } = require("./utilities")
-const manifest = require("../manifest.json")
-const {
-  FIND_HBS_REGEX,
-  FIND_ANY_HBS_REGEX,
-  findDoubleHbsInstances,
-} = require("./utilities")
-const { convertHBSBlock } = require("./conversion")
-const javascript = require("./helpers/javascript")
-const { helpersToRemoveForJs } = require("./helpers/list")
-
-const hbsInstance = handlebars.create()
-registerAll(hbsInstance)
-const hbsInstanceNoHelpers = handlebars.create()
-registerMinimum(hbsInstanceNoHelpers)
-const defaultOpts = {
-  noHelpers: false,
-  cacheTemplates: false,
-  noEscaping: false,
-  escapeNewlines: false,
-  noFinalise: false,
-}
-
-/**
- * Utility function to check if the object is valid.
- */
-function testObject(object) {
-  // JSON stringify will fail if there are any cycles, stops infinite recursion
-  try {
-    JSON.stringify(object)
-  } catch (err) {
-    throw "Unable to process inputs to JSON, cannot recurse"
-  }
-}
-
-/**
- * Creates a HBS template function for a given string, and optionally caches it.
- */
-let templateCache = {}
-function createTemplate(string, opts) {
-  opts = { ...defaultOpts, ...opts }
-
-  // Finalising adds a helper, can't do this with no helpers
-  const key = `${string}-${JSON.stringify(opts)}`
-
-  // Reuse the cached template is possible
-  if (opts.cacheTemplates && templateCache[key]) {
-    return templateCache[key]
-  }
-
-  string = processors.preprocess(string, opts)
-
-  // Optionally disable built in HBS escaping
-  if (opts.noEscaping) {
-    string = exports.disableEscaping(string)
-  }
-
-  // This does not throw an error when template can't be fulfilled,
-  // have to try correct beforehand
-  const instance = opts.noHelpers ? hbsInstanceNoHelpers : hbsInstance
-  const template = instance.compile(string, {
-    strict: false,
+import vm from "vm"
```

Comment:
This will likely break things in Node - might need to require these, hopefully the rollup will convert them back to imports as needed.